mycroft/skills/core.py

```diff
@@ -661,15 +665,27 @@ def make_intent_failure_handler(cls, ws):
         """Goes through all fallback handlers until one returns True"""
 
         def handler(message):
+            # indicate fallback handling start
+            ws.emit(Message("mycroft.skill.handler.start",
+                            data={'handler': "fallback"}))
             for _, handler in sorted(cls.fallback_handlers.items(),
                                      key=operator.itemgetter(0)):
                 try:
                     if handler(message):
+                        #  indicate completion
+                        ws.emit(Message(
+                            'mycroft.skill.handler.complete',
+                            data={'handler': get_handler_name(handler)}))
```

Comment:
The handler name of the complete should match the name given in `mycroft.skill.handler.start` (fallback) Having the name of the actual handler is probably a good idea so I suggest a separate field 'fallback_handler' for the name of the handler.